tests/deprecated.py

```diff
@@ -0,0 +1,26 @@
+from contextlib import contextmanager
+from typing import Type
+
+import pytest
+from packaging import version
+
+import aiogram
+
+
+@contextmanager
+def check_deprecated(
+    max_version: str, exception: Type[Exception], warning: Type[Warning] = DeprecationWarning,
```

Comment:
Is only question: Why exception here without default value?

Reply:
`check_deprecated` is generic function for different features, so I thought passing required exception type makes sense in different cases.

Reply:
tnx